injector/LoggerInstance/AdliLogger.py

```
@@ -143,10 +144,22 @@ def logException(self):
         '''
         self.count += 1
         self.exceptionLogCount += 1
+        
+
+        exc_type, exc_value, tb = sys.exc_info()
```

Ruff (0.14.7)

Unpacked variable `exc_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)